pymongo/encryption_options.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         mongocryptd_spawn_path: str = "mongocryptd",
         mongocryptd_spawn_args: Optional[List[str]] = None,
         kms_tls_options: Optional[Mapping[str, Any]] = None,
+        extra_options: Optional[Mapping[str, Any]] = None,
```

Comment:
AutoEncryptionOpts accepts the spec's "extra_options" as named parameters. So this should be:
```
csfle_path: ...,
csfle_required: ...,
```